src/preamble_minimal.js

```diff
@@ -56,26 +56,32 @@ var HEAP8, HEAP16, HEAP32, HEAPU8, HEAPU16, HEAPU32, HEAPF32, HEAPF64,
 #endif
   wasmMemory, wasmTable;
 
+{{{ (function() {
+  global.assignToModuleIfExportAll = function(x) {
+    return MODULARIZE && EXPORT_ALL ? `Module['${x}'] =` : '';
+  };
+  return null;
+})(); }}}
```

Comment:
See the pattern in `src/library_html5_webgpu.js ` for how to do this without the IIFE